pkg/webui/console/views/gateway-general-settings/index.js

```diff
@@ -42,8 +42,10 @@ import { mayEditBasicGatewayInformation, mayDeleteGateway } from '../../lib/feat
 const m = defineMessages({
   updateSuccess: 'Successfully updated gateway',
   deleteGateway: 'Delete Gateway',
-  modalWarning:
-    'Are you sure you want to delete "{gtwName}"? This action cannot be undone and it will not be possible to reuse the gateway ID and EUI!',
+  modalWarning: `
+Are you sure you want to delete "{gtwName}"? This action cannot be undone and it will not be possible to reuse the gateway ID!
+Note, deleting this gateway will release its EUI and will make it available for registering a new one with the same gateway EUI.
```

Comment:
Do we need to be this explicit?

Reply:
I'd say no. Let's make it consistent with the message for other entities.